recipes/napari-imsmicrolink/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "napari-imsmicrolink" %}
+{% set version = "0.1.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari-imsmicrolink-{{ version }}.tar.gz
+  sha256: d5b35b7980b85d6b24b42b2dc9e8c1de479f6051b128e9d3c69b8a91675054d1
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - aicsimageio
+    - bioformats_jar
+    - czifile
+    - dask-core
+    - h5py
+    - imagecodecs
+    - napari
+    - napari-plugin-engine
+    - numpy
+    - opencv
+    - pandas
+    - python >=3.7
+    - qtpy
+    - simpleitk
+    - tifffile
+    - zarr ==2.10.3
```

Is this the only version known to work with the package @nhpatterson? This can severely restrict the compatibility with other plugins in the napari ecosystem (not to mention that napari itself depends on zarr), so we should consider more relaxed constrains. Maybe like this?

```suggestion
    - zarr >=2.10.3
```